src/test/java/com/netgrif/application/engine/workflow/web/VariableArcsTest.java

```diff
@@ -99,7 +99,7 @@ public void before() throws Exception {
         testHelper.truncateDbs();
         userRunner.run("");
         repository.deleteAll();
-        if (roleRepository.findByName_DefaultValue(ProcessRole.DEFAULT_ROLE) == null) {
+        if (roleRepository.findAllByName_DefaultValue(ProcessRole.DEFAULT_ROLE) == null) {
```

Comment:
why do we have this check?

I belive this should be done with an `assert`ion and in the helper class